wewatch-client/src/components/noMatch/noMatch.js

```diff
@@ -1,29 +1,70 @@
-import React, { useState  } from "react";
+import React, { useState, useEffect  } from "react";
 import { Redirect } from "react-router-dom";
 
+import socket from 'Socket';
+
 import './noMatch.css';
 import Layout from 'views/layout';
 
-const NoMatch = () => {
+const NoMatch = (props) => {
     const [cancel, setCancel] = useState(false);
+    const [tryAgain, setTryAgain] = useState(false);
+    const [tryAgainHost, setTryAgainHost] = useState(false);
+    const [name, setName] = useState(props.location.state.name);
```

Comment:
is setName being used anywhere in this noMatch page? If not should we just have name as a variable in this component? Let me know your thoughts!

Reply:
that makes sense, ill just make it a variable!